core/src/main/java/com/alibaba/nacos/core/remote/circuitbreaker/CircuitBreaker.java

```diff
@@ -115,4 +130,61 @@ public  void loadConfig() {
     public  CircuitBreakerConfig getConfig(String pointName) {
         return pointConfigMap.getOrDefault(pointName, new CircuitBreakerConfig());
     }
+
+    private synchronized void loadRuleFromLocal(String pointName) throws IOException {
+
+        File pointFile = getRuleFile(pointName);
+        if (!pointFile.exists()) {
+            pointFile.createNewFile();
+        }
+        String pointConfigContent = DiskUtils.readFile(pointFile);
+        String monitorKeyContent = DiskUtils.readFile(pointFile);
+
```

Comment:
it seems pointConfigContent and monitorKeyContent are in the same file, and content is consistent?

Reply:
They should be in the separate files. I will fix this in the following commits.